lightkurve/search.py

```diff
@@ -147,12 +147,18 @@ def download(self, quality_bitmask='default', download_dir=None):
                                               download_dir=download_dir)['Local Path']
 
         # return single tpf or lcf
-        tpf_files = ['lpd-targ.fits', 'spd-targ.fits']
-        lcf_files = ['llc.fits', 'slc.fits']
-        if any(file in self.table['productFilename'][0] for file in tpf_files):
+        kepler_tpf_files = ['lpd-targ.fits', 'spd-targ.fits']
+        kepler_lcf_files = ['llc.fits', 'slc.fits']
+        tess_tpf_file = '_tp.fits'
+        tess_lcf_file = '_lc.fits'
+        if any(file in self.table['productFilename'][0] for file in kepler_tpf_files):
             return KeplerTargetPixelFile(path[0], quality_bitmask=quality_bitmask)
-        elif any(file in self.table['productFilename'][0] for file in lcf_files):
+        elif any(file in self.table['productFilename'][0] for file in kepler_lcf_files):
             return KeplerLightCurveFile(path[0], quality_bitmask=quality_bitmask)
+        elif tess_tpf_file in self.table['productFilename'][0]:
+            return TessTargetPixelFile(path[0], quality_bitmask)
+        elif tess_lcf_file in self.table['productFilename'][0]:
+            return TessLightCurveFile(path[0], quality_bitmask=quality_bitmask)
```

Comment:
Can we replace all of the above with `open(self.table['productFilename'][0])` ?

Reply:
Yes, good point! I didn't consider that `open()` takes care of a lot of this functionality.